.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: java
+jdk:
+  - oraclejdk8
```

Comment:
It was my suggestion, if you don't raise the Travis version the planner cannot be built with java 8 tag in maven
El 15/5/2015 4:35 p. m., Andrea Turli <notifications@github.com> escribió:In .travis.yml:
> @@ -1,4 +1,6 @@
>  language: java
> +jdk:
> +  - oraclejdk8

any reason to build the entire SeacloudsPlatform project using jdk8?

—Reply to this email directly or view it on GitHub.